### PR TITLE
fix: change wording on block sync to be more clear

### PR DESCRIPTION
**Pull request by stringhandler**
image

### CODE REVIEW

File: public/locales/en/setup-view.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
         "application-started": "Applications started",
         "downloading": "Downloading",
         "download-completed": "Download completed",
-        "waiting-for-header-sync": "Waiting for header sync. {{local_height}}/{{tip_height}} headers synced",
-        "waiting-for-block-sync": "Waiting for block sync. {{local_height}}/{{tip_height}} blocks synced"
+        "waiting-for-header-sync": "Waiting for header sync. {{local_header_height}}/{{tip_header_height}} headers and {{local_block_height}}/{{tip_block_height}} block synced. ",
```
Comment (brianp) [inline public/locales/en/setup-view.json:21]:
```suggestion
        "waiting-for-header-sync": "Waiting for header sync. {{local_header_height}}/{{tip_header_height}} headers and {{local_block_height}}/{{tip_block_height}} blocks synced. ",
```

File: src-tauri/src/process_watcher.rs [hunk 2]
```diff
@@ -131,11 +131,11 @@ impl<TAdapter: ProcessAdapter> ProcessWatcher<TAdapter> {
                                    }
                                }
                                // Restart dead app
-                               sleep(Duration::from_secs(2)).await;
+                               sleep(Duration::from_secs(1)).await;
                                warn!(target: LOG_TARGET, "Restarting {} after health check failure", name);
                                child.start().await?;
                                // Wait for a bit before checking health again
-                               sleep(Duration::from_secs(10)).await;
+                               sleep(Duration::from_secs(3)).await;
```
Comment (brianp) [inline src-tauri/src/process_watcher.rs:138]: Is this relevant to this PR?

**Review by brianp — CHANGES_REQUESTED** [review]

File: src-tauri/src/process_watcher.rs (hunk 2 above)
Comment (stringhandler) [reply src-tauri/src/process_watcher.rs:138]: lol no

**Comment by Tas4tari** [comment]
Tested successfully on Windows 11, app version 0.5.63

**Comment by Anziskavds** [comment]
Tested successfully on MacBook Air 13 inch M1 2020 on Testnet v0.5.63.

**Pull request merged**